lib/action_policy/graphql/authorized_field.rb

```diff
@@ -23,24 +23,34 @@ def extract_option(key, &default)
 
       class AuthorizeExtension < Extension
         def apply
-          @to = extract_option(:to) { ::ActionPolicy::GraphQL.default_authorize_rule }
-          @raise = extract_option(:raise) do
-            if field.mutation
-              ::ActionPolicy::GraphQL.authorize_mutation_raise_exception
-            else
-              ::ActionPolicy::GraphQL.authorize_raise_exception
-            end
-          end
+          @to = options.delete(:to)
+          @raise = options.delete(:raise)
```

Comment:
@palkan I wasn't able to redefine config options in spec without this trick. Current implementation captures `ActionPolicy::GraphQL` options when the schema is loaded. I think you passed block to `Hash#fetch` in order to make it dynamic but in fact block is executed right away if Hash is missing field.

I haven't update all the other extensions before hearing back from you. But I definitely can do it.

Reply:
> I think you passed block to Hash#fetch in order to make it dynamic

Nope. The idea was to avoid dynamic resolution and use the value available at the schema generation time.

So, we don't need to change it.